sktime/forecasting/base/_base.py

```diff
@@ -1885,6 +1919,9 @@ def _check_fh(self, fh, pred_int=False):
         else:
             # If fh is passed, coerce to ForecastingHorizon and validate (all cases)
 
+            if isinstance(fh, pd.Index):
```

Comment:
Question: is it required? What happens if this block is absent?

Reply:
It is not required (I tested this). I left this code in by accident. This was supposed to be a "docstrings-only" PR. I removed this chunk and also fixed the typo. The goal is not to be exhaustive (too exhausting) but rather to improve the docstring incrementally to include a known omission. Re polars that is a great question but I have no idea (and no experience with polars).